curvefs/src/client/s3/disk_cache_read.cpp

```diff
@@ -171,5 +153,41 @@ int DiskCacheRead::WriteDiskFile(const std::string fileName,
     return writeLen;
 }
 
+int DiskCacheRead::ClearReadCacheByInode(const std::string &inode) {
+    VLOG(1) << "ClearReadCacheByInode start, inode: " << inode;
+
+    // load all read cache
+    std::set<std::string> cachedObj;
+    int ret = LoadAllCacheFile(&cachedObj);
```

Comment:
If there are many diskcache files here, there may be performance problems

Reply:
fixed.